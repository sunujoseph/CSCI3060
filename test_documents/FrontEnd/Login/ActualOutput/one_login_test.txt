Enter Username: 
Error: Already Logged In
