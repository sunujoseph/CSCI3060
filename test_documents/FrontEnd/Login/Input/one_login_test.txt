login
admin1
login
