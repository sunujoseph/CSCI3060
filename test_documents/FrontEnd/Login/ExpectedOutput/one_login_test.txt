Enter Username: 
Error: Already logged in
